Tests/DangerTests/BitBucketServerTests.swift

```diff
@@ -6,26 +6,59 @@ final class BitBucketServerTests: XCTestCase {
     private var bitBucketServer: BitBucketServer {
         bitbucketFixtureDSL.bitbucketServer
     }
-
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/BitBucketServerTests.swift;
  Line: 9;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)